.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - master
+      - develop
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
```

Comment:
Is this means that test will run on different ubuntu version depending on the testing time?

Reply:
It will run on the last version of ubuntu available for actions

Reply:
One test run passed with the current config:

https://github.com/aergoio/aergo/actions/runs/4931682161/jobs/8813966127